int/api/websites/all.go

```diff
@@ -80,24 +74,25 @@ func Registry(client *node.Client, candidateDatastoreKeys [][]byte) ([]*models.R
 		return nil, fmt.Errorf("metadata reaching on dnsContractStorers failed : %w", err)
 	}
 
-	dates := make([]dateOnChain, len(metadatas))
-
-	for index, metadata := range metadatas {
-		var date dateOnChain
-
-		_ = json.Unmarshal(metadata.CandidateValue, &date)
-		dates[index] = date
-	}
+	registryResult := make([]*models.Registry, len(metadatas))
 
-	registryResult := make([]*models.Registry, len(dates))
-	for index, date := range dates {
+	for index := 0; index < len(metadatas); index++ {
 		registryResult[index] = &models.Registry{
-			Name:      strings.Split(recordKeys[index], recordKey)[1],
-			Address:   metadataKeys[index].Address,
-			CreatedAt: time.Unix(date.CreateDate/secondsToMilliCoeff, 0).Format(dateFormat),
-			UpdatedAt: time.Unix(date.UpdateDate/secondsToMilliCoeff, 0).Format(dateFormat),
+			Name:     strings.Split(recordKeys[index], recordKey)[1],
+			Address:  metadataKeys[index].Address,
+			Metadata: ByteArrToString(metadatas[index].CandidateValue),
 		}
 	}
 
 	return registryResult, nil
 }
+
+// transforms the []byte to string.
```

Comment:
The coment is not matching the code. I don't understand why you add a coma between each char...

Reply:
changed